shell/common/shell.cc

```diff
@@ -1341,6 +1346,33 @@ static void ServiceProtocolFailureError(rapidjson::Document* response,
   response->AddMember("message", message, allocator);
 }
 
+bool Shell::OnServiceProtocolDumpSkiaMemory(
+    const ServiceProtocol::Handler::ServiceProtocolMap& params,
+    rapidjson::Document* response) {
+  FML_DCHECK(task_runners_.GetRasterTaskRunner()->RunsTasksOnCurrentThread());
```

Comment:
@chinmaygarde  - should we run this on the platform thread instead and jump to io/raster from there? Or perhaps on IO and jump to raster? I'm not sure if it makes a difference.